### DISCUSSION TITLE
You can install AMS v2.6 or higher on Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by clementgineste** [ant-media/Ant-Media-Server#5051] [Q&A] [answered]
That the output of the installer script with Entreprise zip on a fresh and updated Ubuntu 22.04. yep.

`sudo ./install_ant-media-server.sh -i ant-media-server-enterprise-2.5.3-20230107_0851.zip`
...
`You can install AMS v2.6 or higher on Ubuntu 22.04`

:rocket:

**Comment by yashtandon113** [comment, marked as answer]
@clementgoclock 
Thanks for asking this question here.

AMS v2.5.3 is officially supported on Ubuntu 20.04, while AMS v2.6.0 is officially supported on Ubuntu 22.04. However, v2.6.0 has not yet been officially released; however, you can download the most recent snapshot from your account's download section on https://antmedia.io or we can also send you via email directly.

For v2.5.3, you can use Ubuntu 20.04 if possible.

**Reply by clementgineste** [reply]
Thanks for your answer. I probably misinterpreted this [page](https://antmedia.io/docs/guides/installing-on-linux/installing-ams-on-linux/) when it says that AMS can be installed on Ubuntu 22.04, I thought it was supported by the classic version.
However I wanted to try on Ubuntu 22.04 because I first tried on 20.04 and the installer had an error with a non-existent package, I will open another thread.

**Reply by yashtandon113** [reply]
@clementgoclock 
Thank you for your comments. I'll update the document to clarify this point.

On Ubuntu 20.04 with v2.5.3, there is no such installation problem. You can send us the logs on support@antmedia.io and we'll look them over.
